### PR TITLE
Use the un-underscored name if no alias attribute is there.

### PR DESCRIPTION
**Pull request by jap**
This appears to be caused by a thinko in #391, this fixes it again.

### CODE REVIEW

**Comment by Tinche** [comment]
Ah, cool. Can you add a changelog note? And a simple test would be great so it doesn't happen again.

**Comment by jap** [comment]
Actually -- given that `cattrs` depends on modern `attrs` and this is just compat code, let me simplify it!

**Comment by Tinche** [comment]
Howdy, any progress on this? Trying to plan the next release.

**Comment by jap** [comment]
Whoops, sorry - forgot about this PR, working on an updated version now!

File: tests/test_converter.py [hunk 1]
```diff
@@ -381,6 +381,23 @@ class C:
     assert inst == converter.structure(unstructured, C)
 
 
+def test_dict_roundtrip_with_alias():
+    """
+    A class with an aliased attribute can be unstructured and structured.
+    """
+
+    converter = Converter()
```
Comment (Tinche) [inline tests/test_converter.py:389]: I think you should be testing this with a `BaseConverter` to hit the changed code path, the `Converter` uses different machinery.

**Review by Tinche — CHANGES_REQUESTED** [review]
Looks good, left a comment

**Comment by Tinche** [comment]
Just a small note, I'll be away for the next 3 weeks on a family trip and I can continue reviewing this after I'm back!

**Comment by Tinche** [comment]
I want to get the next release out so I fixed this myself. Thanks!

**Pull request merged**